### PR TITLE
Annika and Pekis Weather App

### PR DESCRIPTION
**Pull request by AnnikaSonnek**
This weather app was built by Annika Sonnek and Aki Sinikoski.

### CODE REVIEW

File: Designs/Design-3/style.css [hunk 1]
```diff
@@ -0,0 +1,422 @@
...
+
+
+
+
+
+
+/* BIGGER MOBILE        BIGGER MOBILE        BIGGER MOBILE */
+ 
+
+@media (min-width: 380px) {
```
Comment (AntoniaGranit) [inline Designs/Design-3/style.css:211]: It looks really good on mobile! I like the way the shadowed frame disappears and it's only the content inside the container. Very sleek!

**Review by AntoniaGranit — APPROVED** [review]

**Comment by AntoniaGranit** [comment]
Really great and sleek styling and HTML! It looks great in mobile and (even though I'm not reviewing the javascript file) the Javascript seems to work well, sans the light/dark mode feature. I really like the icons and the font. Very impressive CSS!

File: Designs/Design-3/style.css [hunk 2]
```diff
@@ -0,0 +1,422 @@
...
+    --dark-background: #232323;
+    --dark-primary: #FBF8F3;
+    --dark-link: #A9FE75;
+    --dark-filter: invert(100%) sepia(60%) saturate(759%) hue-rotate(293deg) brightness(108%) contrast(97%);
+    --dark-box-shadow: 5px 7px 41px 9px rgba(255,255,255,0.3),6px 6px 25px -10px rgba(255,255,255,0.3);
+    --light-background: #FBF8F3;
+    --light-primary: #232323;
+    --light-link: #FF5733;
+    --light-filter: invert(10%) sepia(0%) saturate(446%) hue-rotate(138deg) brightness(98%) contrast(91%);
+    --light-box-shadow: 5px 7px 41px 9px rgba(0,0,0,0.1),6px 6px 25px -10px rgba(0,0,0,0.1);
```
[the remark this answers is not in the document]
Comment (AnnikaSonnek) [reply Designs/Design-3/style.css:21]: Yeah, it was kind of a bummer that we weren't able to implement the dark and light mode. If we hade more time I think we would have been able to make it work.

Peki is the one who should have all the credit for the styling on this project! ^_^

**Review by AjmalZ — COMMENTED** [review]
The JS code is clean and well structured, very easy to follow. The whole project overall is looking good. Well done!

**Pull request closed without merging**